### DISCUSSION TITLE
GraphQL OR operator

### DISCUSSION THREAD

**Discussion by patryk-kawalarz** [directus/directus#10374] [Archive]
How can I get posts where the date is less than now OR is empty?

**Comment by rijkvanzanten** [comment]
You can use the logical _or operation in the filter attribute. The following should do the trick:

```
filter: {
  _or: [
    { date: { _lt: "$NOW" } },
    { date: { _null: true } }
  ]
}
```

(assuming your field is called `date`)

**Reply by patryk-kawalarz** [reply]
Thank you very much. My field is a datetime type. And I have an error when I use $NOW here:
Error from global error handler Error: Method Date.prototype.toString called on incompatible receiver [object Object]

**Reply by rijkvanzanten** [reply]
This sounds like a (potential) bug. Mind [opening a new issue](https://github.com/directus/directus/issues/new)? Thanks!
